### PR TITLE
fix cloud snaps page as per PD-176

### PR DESCRIPTION
**Pull request by andreipope**
fix cloud snaps page as per PD-176

### CODE REVIEW

**Comment by andreipope** [comment]
@sanjaynaikwadi @harsh-px please review the changes when you have some time.

File: content/reference/CLI/cloud-snaps.md [hunk 1]
```diff
@@ -54,11 +54,39 @@ Flags:
   -h, --help   help for cloudsnap
 ```
 
-### Set the required cloud credentials
+### Login to the secrets database
+
+Note that the cloud credentials are stored in an external secret store. Hence, before creating the credentials, make sure that you have [configured a secret provider](/key-management) of your choice.
+
+Now that the secret store has been configured, we can list the configured credentials:
```
Comment (talakad) [inline content/reference/CLI/cloud-snaps.md]: This probably should move to later point once we have described to create a credential OR clarify that we are listing any existing credentials before creating one. and also clarify that listing credential does mean that connection to secret-store end point has been validated.

File: content/reference/CLI/cloud-snaps.md [hunk 2]
```diff
@@ -120,37 +148,41 @@ If you are using _AWS_, _Portworx_ creates a bucket (`ID` same as the cluster `U
...
+##### With a user-specified bucket (applicable only from 1.5.0 onwards)
+
+Say you are using `us-east-1 region`. If so, you should type something like the following:
 
 ```text
 pxctl credentials create --provider s3  --s3-access-key AKIAJ7CDD7XGRWVZ7A --s3-secret-key mbJKlOWER4512ONMlwSzXHYA --s3-region us-east-1 --s3-endpoint s3.amazonaws.com --bucket bucket-id my-s3-cred
 ```
 
+If you are using a different region, replace the `--s3-region` and `--s3-endpoint` parameters with the appropriate values. For more information about region-specific endpoints, check out the "Amazon Simple Storage Service (Amazon S3)" section on [this page](https://docs.aws.amazon.com/general/latest/gr/rande.html).
+
```
Comment (talakad) [inline content/reference/CLI/cloud-snaps.md]: This is nice!. Thanks. With 2.1.2 and 2.2, there is new parameter "disable-path-style" that should be specified while creating credential for  those s3 providers that do not support path-style and support only dirt-host-style.

Comment (andreipope) [reply content/reference/CLI/cloud-snaps.md]: @talakad Just wondering, can someone help with the actual command and the output? I don't have an environment to test it out.

File: content/reference/CLI/cloud-snaps.md (hunk 1 above)
Comment (andreipope) [reply content/reference/CLI/cloud-snaps.md]: > This probably should move to later point once we have described to create a credential OR clarify that we are listing any existing credentials before creating one. and also clarify that listing credential does mean that connection to secret-store end point has been validated.

Makes sense. It is now moved.

File: content/reference/CLI/cloud-snaps.md (hunk 2 above)
Comment (talakad) [reply content/reference/CLI/cloud-snaps.md]: if you try to create credentials for an s3 endpoint that supports only virt-host-style  access, then you will hit error like below
"createCred: error validating credential during create:
 SecondLevelDomainForbidden: Please use virtual hosted style to access.
    status code: 403, request id: xxyyzzaabbcc, host id:",
in this case specify --disable-path-style parameter while creating credentials. For example;
"pxctl cred create mycreds --provider=s3 --s3-disable-ssl --s3-region=us-east-1 --s3-access-key=<S3-ACCESS_KEY> --s3-secret-key=<S3-SECRET_KEY> --s3-endpoint=mys3-enpoint.com --disable-path-style --bucket=mybucket
Credentials created successfully, UUID:77c336ac-9937-46cf-ad42-297ea41c8022"

Comment (andreipope) [reply content/reference/CLI/cloud-snaps.md]: > if you try to create credentials for an s3 endpoint that supports only virt-host-style access, then you will hit error like below
> `createCred: error validating credential during create: SecondLevelDomainForbidden: Please use virtual hosted style to access. status code: 403, request id: xxyyzzaabbcc, host id:`,
> in this case specify --disable-path-style parameter while creating credentials. For example;
> `pxctl cred create mycreds --provider=s3 --s3-disable-ssl --s3-region=us-east-1 --s3-access-key=<S3-ACCESS_KEY> --s3-secret-key=<S3-SECRET_KEY> --s3-endpoint=mys3-enpoint.com --disable-path-style --bucket=mybucket Credentials created successfully, UUID:77c336ac-9937-46cf-ad42-297ea41c8022`

Thank you so much for the details. It is now updated.

**Pull request merged**